### PR TITLE
Drop-in would throw undefined error when updating payment method configuration

### PR DESCRIPTION
**Pull request by Rhymond**
### Summary
Fixed issue where undefined error was thrown when payment method is not visible but is set in configuration.

### Checklist

- [x] Added a changelog entry

### CODE REVIEW

File: src/dropin.js [hunk 1]
```diff
@@ -383,6 +383,10 @@ Dropin.prototype.updateConfiguration = function (property, key, value) {
     return;
   }
 
+  if (this._mainView.getView(property) === undefined) {
```
Comment (crookedneighbor) [inline src/dropin.js]: I think this should be:

```js
var view = this._mainView.getView(property);

if (!view) {
  return;
}

view.updateConfiguration(key, value);
```

**Comment by Rhymond** [comment]
This situation occurs at the moment for me then we're trying to update apple pay configuration after price change. If user don't use device which is capable to work with Apple pay, we get `undefined` error. 

I would agree with more descreptive error message, but in any case it's not an error more like warning by my case.

Let me know what do you think it would be correct behaviour and I will update code respectively on your decision.

File: src/dropin.js (hunk 1 above)
Comment (crookedneighbor) [reply src/dropin.js]: This should still be updated.

**Comment by Rhymond** [comment]
Thanks for an answer. 

Yup, that's right. I believe PayPal will work on almost all cases, but we have an issue especially with Apple Pay. 

I'll fix current PR and will push changes for a review.

**Comment by crookedneighbor** [comment]
This needs a test update to cover this case, but we can go ahead and do that once it's merged in if you're not comfortable updating the unit tests.

**Comment by Rhymond** [comment]
@crookedneighbor I will try to update unit tests over the weekend

**Comment by crookedneighbor** [comment]
@Rhymond How's it coming? Do you need us to handle the tests?

**Comment by Rhymond** [comment]
@crookedneighbor sorry, had no time over the weekend. If you can would be great, if you can wait for one more week I can try to do it. 
Thanks.

**Comment by crookedneighbor** [comment]
Merged in with https://github.com/braintree/braintree-web-drop-in/commit/8369e45fba7a003299ce5c2038795da93cd6c417

**Pull request closed without merging**